docs/topics/class-based-views/mixins.txt

```diff
@@ -260,7 +261,8 @@ We can hook this into our URLs easily enough:
     :caption: ``urls.py``
 
     from django.urls import path
-    from books.views import RecordInterestView
+
+    from .views import RecordInterestView
```

Comment:
Leave alone. This looks like a root URLConf.

Reply:
ok